### PR TITLE
feat: update readme

### PR DESCRIPTION
**Pull request by ndtang9**

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -12,40 +12,46 @@ An AI Application that generates diagram from user system requirements
...
-    - on Intranet 
-- The design output must consider the security, maintainability, scalability,...
+-   Generate System Architecture from User requirements
+-   Follow AWS Well Architecture
+-   Can export result to **drawio** or **Mermaid**
+-   Support **justification/explanation** where user can add more input from first result
+-   Leverage the power of generative AI and
+    -   open knowledge on Internet (for example: architecture blueprint shared by AWS, etc.)
+    -   on Intranet
+-   The design output must consider the security, maintainability, scalability,...
```
Comment (ndtang9) [inline readme.md:24]: This is autoformat changed

File: readme.md [hunk 2]
```diff
@@ -12,40 +12,46 @@ An AI Application that generates diagram from user system requirements
...
 ### `3` Export result to draw.io or mermaid format
+
 ### `4` Generate explanation/document for diagram
-### `5` Generate code templates (Terraform) based on design
-### `6` Import document
+
+### `5` Generate code templates (Terraform) able to run plan based on design
+
+### `6` Import knowledge base
+
```
Comment (ndtang9) [inline readme.md:47]: This is autoformat changed

File: readme.md [hunk 3]
```diff
@@ -54,20 +60,20 @@ An AI Application that generates diagram from user system requirements
...
+-   [hdduytran](https://github.com/hdduytran)
 
 [image-banner]: https://github.com/buisihung11/SyArGPT/blob/main/assets/banner.png?raw=true
 [component-arch]: https://github.com/buisihung11/SyArGPT/blob/main/assets/ComponentArchitect.png?raw=true
 [deployment-link]: https://syargpt.vercel.app
 [system-arch]: https://github.com/buisihung11/SyArGPT/blob/main/assets/system-architecture.png?raw=true
 [general-process]: https://github.com/buisihung11/SyArGPT/blob/main/assets/general-process.png?raw=true
-[app-screenshot]: https://github.com/buisihung11/SyArGPT/blob/main/assets/app-screenshot.jpeg?raw=true
\ No newline at end of file
+[app-screenshot]: https://github.com/buisihung11/SyArGPT/blob/main/assets/app-screenshot.jpeg?raw=true
```
Comment (ndtang9) [inline readme.md:79]: This is autoformat changed

**Comment by buisihung11** [comment]
Lgtm

**Pull request merged**